### PR TITLE
Update Docs for 0.11

### PR DESCRIPTION
**Pull request by timgrossmann**
#### This PR only handles the pure Documentation changes, source code changes will be done in separate PR (relating to issue #1736 )

- [x] Login in with Bosch ID under Pre-requisites in tutorials

- [x] Pre-requisite: You are collaborator / owner of a namespace

- [x] Role Matrix
    - [x] Remove Edit Namespace and replace with Add / Remove Namespace
    - [x] Change Label from 'Submit Models for Review' to 'Initiate Release process'
    - [x] Change Label from 'Release Models' to 'Approve Releases'

- [x] Make matrix vertical alignment

- [x] Add 'Deprecated models' which can only be done by promoter role

- [x] Update Documentation about Model States "https://github.com/eclipse/vorto/blob/development/repository/docs/model_states.md" and link from gettting started and other related pages

### CODE REVIEW

**Comment by timgrossmann** [comment]
#1736

File: docs/tutorials/managing_collaborators.md [hunk 1]
```diff
@@ -30,11 +30,17 @@ Each Role gives the collaborator permissions for specific actions for the accord
...
-| CREATE   | ✔           | ✔             | ✔           | ✔             | ✘                        | ✘              | ✘              | ✘              | ✘                 | ✘                    | ✘                         |
-|  PROMOTE | ✔           | ✘             | ✘           | ✘             | ✔                        | ✘              | ✘              | ✘              | ✘                 | ✘                    | ✘                         |
-| REVIEW   | ✔           | ✘             | ✘           | ✘             | ✘                        | ✔              | ✘              | ✘              | ✘                 | ✘                    | ✘                         |
-| PUBLISH  | ✔           | ✘             | ✘           | ✘             | ✘                        | ✘              | ✔              | ✘              | ✘                 | ✘                    | ✘                         |
-| ADMIN    | ✔           | ✘             | ✘           | ✘             | ✘                        | ✘              | ✘              | ✔              | ✔                 | ✔                    | ✔                         |
+|                           | VIEW | CREATE |  PROMOTE | REVIEW | PUBLISH | ADMIN |
```
Comment (aedelmann) [inline docs/tutorials/managing_collaborators.md]: What about changing ADMIN to 'MANAGE' ? :)

**Comment by aedelmann** [comment]
@timgrossmann 
model.states.md linking to wrong heading. Travis CI build fails coz of that.can u check ?

**Comment by timgrossmann** [comment]
@aedelmann Ready for review

**Pull request merged**